From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
Date: Sat, 28 Jan 2023 14:10:43 -0500
Subject: [PATCH] Dont close Slime Plugin Classloader


diff --git a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
index 12449e18180d604e9cbbc744da74a8b222a18e1f..f770c2c5f77486dc1abe9bc33c9831b8f7ebf362 100644
--- a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
+++ b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
@@ -269,8 +269,10 @@ public final class PluginClassLoader extends URLClassLoader implements io.paperm
             // Paper end
             super.close();
         } finally {
+            if (this.plugin == null || !this.plugin.getName().equals("SlimeWorldManager")) { // ASWM - Don't close
             this.closed = true; // Pufferfish
             jar.close();
+            } // ASWM - Don't close
         }
     }
 
