From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: vytskalt <vytskalt@protonmail.com>
Date: Mon, 16 Oct 2023 18:28:10 +0300
Subject: [PATCH] fix compile


diff --git a/src/main/java/com/infernalsuite/aswm/level/SlimeLevelInstance.java b/src/main/java/com/infernalsuite/aswm/level/SlimeLevelInstance.java
index 2d18d76829b6dc590913d974d50dbaafcb79e175..f31d5f5e6f9bebdc91354491b3bfa40d7f37f809 100644
--- a/src/main/java/com/infernalsuite/aswm/level/SlimeLevelInstance.java
+++ b/src/main/java/com/infernalsuite/aswm/level/SlimeLevelInstance.java
@@ -25,6 +25,7 @@ import net.minecraft.util.Unit;
 import net.minecraft.util.datafix.DataFixers;
 import net.minecraft.world.Difficulty;
 import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.level.ChunkPos;
 import net.minecraft.world.level.biome.Biome;
 import net.minecraft.world.level.chunk.ChunkAccess;
 import net.minecraft.world.level.chunk.ChunkGenerator;
@@ -183,7 +184,7 @@ public class SlimeLevelInstance extends ServerLevel {
                                     .map((tag) -> (net.minecraft.nbt.CompoundTag) Converter.convertTag(tag))
                                     .collect(Collectors.toList()), this)
                             .toList()
-            ));
+            ), new ChunkPos(chunkX, chunkZ));
         }
     }
 
