From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul19988 <paul@paulcodes.co.uk>
Date: Sat, 12 Aug 2023 22:07:44 +0100
Subject: [PATCH] Build Changes


diff --git a/build.gradle.kts b/build.gradle.kts
index 6839e7779cf81b728433eea1e7c3e2ef5e1b00a1..0f8c03f53492cd86b7998e090c933d2db87f4a17 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -14,6 +14,7 @@ val alsoShade: Configuration by configurations.creating
 
 dependencies {
     implementation(project(":slimeworldmanager-api")) // ASWM // Pufferfish // Paper
+    implementation(project(":core")) // ASWM
     // Pufferfish start
     implementation("io.papermc.paper:paper-mojangapi:1.19.2-R0.1-SNAPSHOT") {
         exclude("io.papermc.paper", "paper-api")
