From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: vytskalt <vytskalt@protonmail.com>
Date: Mon, 1 Jan 2024 15:50:10 +0200
Subject: [PATCH] Build Changes


diff --git a/build.gradle.kts b/build.gradle.kts
index 9dfcb42e34221f427a65dbfdce500bd1aa142c7d..ab9f3d8f215394368c81a36c81766cc4ce3768ff 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -13,7 +13,8 @@ configurations.named(log4jPlugins.compileClasspathConfigurationName) {
 val alsoShade: Configuration by configurations.creating
 
 dependencies {
-    implementation(project(":pufferfish-api")) // Pufferfish // Paper
+    implementation(project(":slimeworldmanager-api")) // ASWM // Pufferfish // Paper
+    implementation(project(":core")) // ASWM
     // Pufferfish start
     implementation("io.papermc.paper:paper-mojangapi:1.19.2-R0.1-SNAPSHOT") {
         exclude("io.papermc.paper", "paper-api")
